### DISCUSSION TITLE
Change edge thickness

### DISCUSSION THREAD

**Discussion by gui1ei** [xyflow/xyflow#1551] [Q&A]
It's there any way to change the 'thickness' of the edge dynamically?
In my draw, the edges are representing a number, so I want to change the 'thickness' of the edge by this number. 
In the examples, I didn't find an easy way to do that, because there is no prop for that.

**Comment by moklick** [comment]
You can pass a style attribute to an edge: `style: { strokeWidth: 4 }` https://reactflow.dev/docs/api/edges/
